extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -101,7 +101,7 @@ $start-a-wikia-logo-size: 30px;
 		}
 	}
 
-	@media #{$breakpoint-small-only} {
+	@media #{$breakpoint-below-small-plus} {
```

Comment:
I'm all against it - `breakpoints-layout.scss` is only for site-wide layout changes (like font and margins) - Here we have one extension that looks different in dependence from breakpoints - ultimately all code from `breakpoint-*` files should be moved to proper extensions.